### DISCUSSION TITLE
extracting the number of cells that are considered doublets

### DISCUSSION THREAD

**Discussion by tanasa** [GreenleafLab/ArchR#1359] [Questions / Documentation]
Dear all, 

Would you please advise how shall I extract the number of potential dublets  in a sample (ie. given a list of cells in the sample to assign to each cell the labels "doublet / no dublet"). Thanks a lot, 

Bogdan

**Comment by rcorces** [comment]
Thats a bit of an odd question. Normally, you run `addDoubletScores()` and then `filterDoublets()` which removes all of the doublets from your project based on the provided params. So I guess you would be asking how to identify which cells are doublets before running `filterDoublets()` but after running `addDoubletScores()`? You could relatively easily step through the code in `filterDoublets()` to get this answer.
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/FilterCells.R#L45

**Reply by tanasa** [reply]
Thank you Ryan. 

Just for my own information please, as I have been using ArchR only for a short while.

Assuming that we do not use ArchR or any other dublet scoring algorithm, what is the typical range of median number of fragments per cell that we shall see in a successful experiment ? 

For example, shall we  have an experiment where the median_number_of_fragments per cell is ~ 90 000, would it indicate dublets or rather triplets ? Thanks a lot !

**Reply by rcorces** [reply]
Number of fragments is a very poor predicter of doublets. We showed this in Fig. 1h of the ArchR manuscript:
![image](https://user-images.githubusercontent.com/10343046/160938742-6bb3a7f9-cae2-426b-8645-86692561b76b.png)
